HICPluginTests/Integration/ReflectionToDatabaseTester.cs

```diff
@@ -58,7 +58,7 @@ public void SendTooLongFieldToDatabase()
         try
         {
             var ex = Assert.Throws<Exception>(() => ReflectionBasedSqlDatabaseInserter.MakeInsertSqlAndExecute<TestObject>(t, con, dbInfo, "TestObject"));
-            Assert.IsTrue(ex?.Message.Contains("Field1 in table TestObject is defined as length  10 in the database but you tried to insert a string value of length 41"));
+            Assert.That(ex?.Message.Contains("Field1 in table TestObject is defined as length  10 in the database but you tried to insert a string value of length 41"),Is.True);

```

Comment:
There's a large space between length and 10, is that meant to be like that?

Reply:
Looks like the way a double space in the exception message from a /loong/ time ago. Good catch!